.github/workflows/backport.yml

```diff
@@ -0,0 +1,31 @@
+name: Backport merged pull request
+on:
+  pull_request_target:
+    types: [closed]
+  issue_comment:
+    types: [created]
+permissions:
+  contents: write # so it can comment
+  pull-requests: write # so it can create pull requests
+jobs:
+  backport:
+    name: Backport pull request
+    runs-on: ubuntu-latest
+
+    if: >
+      (
+        github.event_name == 'pull_request_target' &&
+        github.event.pull_request.merged
+      ) || (
+        github.event_name == 'issue_comment' &&
+        github.event.issue.pull_request &&
+        github.event.comment.user.id != 97796249 &&
+        startsWith(github.event.comment.body, '/backport')
+      )
+    steps:
+      - uses: actions/checkout@v4
+      - name: Create backport pull requests
+        uses: korthout/backport-action@v2
```

Comment:
Adding GitHub Actions like this is a security risk as they get GitHub access tokens. This action isn't well known and seems to be maintained by a single person. I would advise against using it in this fashion. In fact, I would strongly prefer not to introduce any more GitHub Actions as we should actually lock down the org to prevent unintended actions from being added.